plone/dexterity/content.py

```diff
@@ -354,22 +354,20 @@ def _set__name__(self, value):
     __name__ = property(_get__name__, _set__name__)
 
     def UID(self):
-        """Returns the item's globally unique id."""
+        # Returns the item's globally unique id.
         return IUUID(self)
 
     @security.private
     def notifyModified(self):
-        """Update creators and modification_date.
-
-        This is called from CMFCatalogAware.reindexObject.
-        """
+        # Update creators and modification_date.
```

Comment:
No. So technically this change is unneeded.
But:
1. This sets a good example.
2. This avoids problems when someone later changes it to protected or public, here or in a subclass.
3. I don't want to think about it. There is no need for this to be publishable, so it should not get a doc string.
4. Current personal mode: kill all doc strings. ;-)